src/components/ComposerQuill/utils.js

```diff
@@ -57,7 +57,7 @@ export function getQuillText(quill) {
       // convert '<' and '>' characters to their html entities instead
       // we don't want to mix html tags the user sent with our conversion of markdown to html
       // replace '<' to '&lt;', and replace '>' NOT at the beginning of a line to '&gt;'
-      sanitized += op.insert.replace(/</g, '&lt;').replace(/(?<!^)>/gm, '&gt;');
+      sanitized += op.insert.replace(/</g, '&lt;').replace(/(?!^)>/gm, '&gt;');
```

Comment:
Lookbehind isn't support by safari yet. After googling around, lookahead is supported by all browsers so used that instead. 
https://stackoverflow.com/questions/641407/javascript-negative-lookbehind-equivalent/11347100#11347100